VideoGame_Beginning

```diff
@@ -0,0 +1,17 @@
+Resaons for Argument
+
+Dataset
+
+Questions
+
+Theory
+
+Explanation
```

Comment:
Video games have increased total global players, however have not retained a total global player increase for titles as time goes on. Only a increase in user scores have increased consistently over time where total global players for a title do not increase consistently by year.